resources/lang/en/model_validation.php

```diff
@@ -88,11 +88,28 @@
         'invalid_email' => "Doesn't seem to be a valid email address.",
         'too_short' => 'New password is too short.',
         'unknown_duplicate' => 'Username or email address already used.',
+        'username_available_in_days' => 'This username will be available for use in <strong>:count</strong> day.|This username will be available for use in <strong>:count</strong> days.',
+        'username_available_in_hours' => 'This username will be available for use in <strong>:count</strong> hour.|This username will be available for use in <strong>:count</strong> hours.',
```

Comment:
The `<strong>` part can probably be extended to include its unit as well (or just removed because current design has `<strong>` wrapping the whole thing). And then the hours and days version can be merged or something.